disBatch.py

```diff
@@ -1,1524 +1,18 @@
 #!/usr/bin/env python3
 
-import json, logging, os, random, re, signal, socket, subprocess as SUB, sys, time
-
-from ast import literal_eval
-from collections import defaultdict as DD
-
-try:
-    from queue import Queue, Empty
-except ImportError:
-    from Queue import Queue, Empty
-from threading import Thread
-
-DisBatchRoot = os.environ.get('DISBATCH_ROOT', None)
-DbUtilPath = '?Not Set Yet?'
-
-if DisBatchRoot:
-    DisBatchPath = DisBatchRoot + os.path.sep + 'disBatch.py'
-    ImportDir = DisBatchRoot
-else:
-    # Try to guess.
-    DisBatchPath = os.path.realpath(__file__)
-    ImportDir = os.path.dirname(DisBatchPath)
-
-sys.path.append(ImportDir)
+import os, sys
 
+dr = None
```

Comment:
```python3
dr = os.getenv('DISBATCH_ROOT')
if dr and dr not in sys.path:
    sys.path.append(dr)
```
is a bit cleaner, but obviously both work fine

Reply:
Yes, cleaner. Thanks.

What's the typical workflow now. I make this change (and the other changes you've suggested), commit and push? Or is there something more streamlined I could do via this review interface?

Reply:
The typical workflow would be to make the changes suggested below, commit and push them to this branch. The GUI will reflect the changes.